sdk/storage/src/queue/requests/get_messages_builder.rs

```diff
@@ -107,55 +90,43 @@ where
     }
 }
 
-impl<'a, C> TimeoutSupport for GetMessagesBuilder<'a, C>
+// methods callable only when every mandatory field has been filled
+impl<'a, C> GetMessagesBuilder<'a, C>
 where
     C: Client + Clone,
 {
-    type O = Self;
+    pub fn queue_client(&self) -> &'a QueueClient<C> {
+        self.queue_client
+    }
```

Comment:
I hope like in #111 we can get rid of these getter methods. They're not really needed.

Reply:
Yes I wholeheartedly agree 👍 👍 !